### DISCUSSION TITLE
ClusterAPI for AWS Prerequisites

### DISCUSSION THREAD

**Discussion by d3bt3ch** [kubernetes-sigs/cluster-api#8947] [Q&A] [answered]
What are the prerequisites for Cluster API for AWS. I am trying all default values but my cluster describe stocks at `WaitingForKubeadmInit`. I need a good documentation for creating the management cluster. I guess I am doing something wrong setting up the management cluster with `aws provider`.

Also what permissions are required for the `aws user` used as the environment variable when initialising the `aws provider`?

**Comment by sbueringer** [comment, marked as answer]
This should all be documented in the quickstart: https://cluster-api.sigs.k8s.io/user/quick-start.html

Apart from that you probably get better answers in the cluster-api-provider-aws repository or Slack channel
